samples/modules/canopennode/README.rst

```diff
@@ -394,48 +394,33 @@ Building and Running for FRDM-K64F
 The sample can be rebuilt with MCUboot and program download support
 for the FRDM-K64F as follows:
 
-#. Build and flash MCUboot by following the instructions in the
-   :ref:`mcuboot` documentation page.
-
-#. Rebuild the CANopen sample with MCUboot support:
+#. Build the CANopenNode sample with MCUboot support:
 
    .. zephyr-app-commands::
-      :zephyr-app: samples/modules/canopennode
+      :tool: west
+      :app: samples/modules/canopennode
       :board: frdm_k64f
       :goals: build
-      :gen-args: -DCONFIG_BOOTLOADER_MCUBOOT=y
+      :west-args: --sysbuild
       :compact:
 
-#. Sign the newly rebuilt CANopen sample binary (using either the
-   demonstration-only RSA key from MCUboot or any other key used when
-   building MCUboot itself):
-
-   .. code-block:: console
-
-      west sign -t imgtool --bin --no-hex -- --key mcuboot/root-rsa-2048.pem \
-              --version 1.0.0
-
-#. Flash the newly signed CANopen sample binary using west:
+#. Flash the newly built MCUboot and CANopen sample binaries using west:
 
    .. code-block:: console
 
-      west flash --skip-rebuild --bin-file zephyr/zephyr.signed.bin
+      west flash --skip-rebuild
 
 #. Confirm the newly flashed firmware image using west:
 
    .. code-block:: console
 
-      west flash --skip-rebuild --runner canopen --confirm-only
+      west flash --skip-rebuild --domain canopennode --runner canopen --confirm-only
 
-#. Finally, resign the CANopen sample binary with a new version number
-   and perform a program download over CANopen:
+#. Finally, perform a program download via CANopen:
```

Comment:
This would load the same application to it, so slot 0 == slot 1, the version and hash would be the same, is this desired?

Reply:
Yes. This is simply for demonstrating how to perform the program download.